addon/components/tooltip-on-element.js

```diff
@@ -67,8 +67,11 @@ export default EmberTetherComponent.extend({
   'aria-hidden': computed.not('tooltipIsVisible'),
   attributeBindings: ['aria-hidden', 'role', 'tabindex'],
   classNameBindings: ['effectClass'],
-  classPrefix: 'tooltip',
-  classNames: ['tooltip'],
+  classPrefix: 'ember-tooltip',
+  classNames: ['ember-tooltip', 'tooltip'],
```

Comment:
> @miguelmadero why would we need "two classes for postitionClass, effectClass and typeClass for the same reasons...."?

That's not relevant anymore since it's simpler to have a single class and do it as a single release. *If* we were to do it in two releases and have 2 classes for the first release so it doesn't break in case someone is styling `tooltip`, then following the same principle, we would have to maintain two classes for `postionClass` et.al because someone might be installing `tooltip-target-attached-left` which is changing to `ember-tooltip-target-attached-left`. 

Anyway, none of this matter since it's simpler to have a single release.